app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -22,11 +22,29 @@
 
   function SearchFilterCtrl($scope, dimStoreService, $timeout, $interval) {
     var vm = this;
+    var filterInputSelector = '#filter-input';
+
+    $scope.search = {
```

Comment:
Set controller properties on 'vm' because that goes on the controller instead of scope.  This keeps scope clean.